### PR TITLE
Quick Setup Script Fixes

### PR DESCRIPTION
**Pull request by enjeyw**
**Describe the Pull Request**

Fixed:
- Quick setup script didn't work if you don't use postgress, password as postgres credentials.
- Some readme stuff around which directories to run various commands
- Some directory issues in scripts

**Todo**

- [ ] Link relevant [trello card](https://trello.com/b/PA2LhlOh/sempo-dev) or [related issue](https://github.com/teamsempo/SempoBlockchain/issues) to the pull request
- [ ] Request review from relevant @person or team
- [ ] QA your pull request. This includes running the app, login, testing changes etc.
- [ ] Bump [backend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/config.py#L7) and/or [frontend](https://github.com/teamsempo/SempoBlockchain/blob/e3eb0f480e86d5a8ef2c1814127b70ff018671c4/app/package.json#L3) versions following Semver

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,18 +40,21 @@ npm install -g ganache-cli@6.4.1
...
 ```
 
 **Front-End**
 
 Our frontend uses react.
 
+from app directory:
+
 ```
-cd app
```
Comment (enjeyw) [inline README.md:49]: this was too dependent on guessing which directory user was in. Makes more sense to just take `cd` as assumed knowledge and let the user get to the right dir

File: README.md [hunk 2]
```diff
@@ -35,18 +40,21 @@ npm install -g ganache-cli@6.4.1
 
 Download and install python 3.6 and its respective pip and virtualenv (**python 3.7 will break things**). Then:
 
+from devtools directory:
+
 ```
 python3 -m venv venv
 source venv/bin/activate
-./devtools/install_python_requirements.sh
```
Comment (enjeyw) [inline README.md:41]: keep a consistent pattern of 'just run the script from the dir'

Comment (michielderoos) [reply README.md:41]: Along these lines: The invoke_tests script doesn't seem to work any more

Comment (enjeyw) [reply README.md:41]: hmmm what error are you getting @michielderoos ?

Comment (michielderoos) [reply README.md:41]: Figured it out, it was just a problem with my local database!

**Pull request merged**